src/main/java/service/impl/GetValueForFruitImpl.java

```diff
@@ -0,0 +1,40 @@
+package service.impl;
+
+import dao.FruitDao;
+import dao.FruitDaoImpl;
+import service.GetValueForFruit;
+
+public class GetValueForFruitImpl implements GetValueForFruit {
+    private static final GetOperationTypeImpl getOperationTypeImpl = new GetOperationTypeImpl();
+    private static final GetFruitFromFileImpl getFruitFromFileImpl = new GetFruitFromFileImpl();
+    private static final String COMMA = ",";
+    private static final int TYPE_FRUIT_INDEX = 0;
+    private static final int FRUIT_INDEX = 1;
+    private static final int VALUE_INDEX = 2;
+
+    private final FruitDao fruitDao = new FruitDaoImpl();
+
+    @Override
+    public Integer calculateNewValue(String line) {
+        String[] arr = line.split(COMMA);
```

Comment:
arr is a bad naming for array, like list for a List<> and string for a string - don't name variables after a data type, name them after what they store/what they are used for